### ISSUE TITLE
Setting to disable the automatic use of cover images from videos

### ISSUE THREAD

**Issue by SubJunk** [UniversalMediaServer/UniversalMediaServer#1073]
I think it would be a good option to add since it looks weird when you have 9/10 videos with generated thumbnails and then one has a cover image. I'd like the option to make it consistent so all of them are generated.

**Comment by Sami32** [comment]
Oh, i think i understand what you mean now.
It also generated error with MediaInfo > 7.81 on WinXP, it's why i stick to 7.81 version.

Do you really want make it in the GUI ?
I think that should be a non-GUI parameter disabling the use of internal coverart by default.

**Comment by Nadahar** [comment]
I'm copying my reply in #1070 here:

Regarding thumbnails we have different sources for all media types. All the enable/disable options and the outcome of their combinations is confusing to me, so I guess it is to most users as well. I've been thinking of having an editable priority list for each type (audio, video, images) instead. Generic icons would be one of those choices, file or folder images, embedded images and downloaded would be other choices. A reasonable default working well for most should be found, but those that have different priorities could make those in a much more predictable way than how it is now.

In that same priority list categories could also be enabled/disabled for those that never want a certain type to be used. It would be much like the engine selection is today.

**Comment by Sami32** [comment]
@Nadahar Not sure i understood you.
Do you mean making the already existing non-GUI parameters (audio, video, audio) in the GUI as well ?

**Comment by Nadahar** [comment]
Yes and no. I'm saying, remove all the different options both in GUI and not controlling thumbnail generation, set up a list of possible thumbnail sources for each category (audio, video, image) and let the user edit the priority and enable/disable sources. E.g:

Audio thumbails |
------------------|
* Embedded coverart|
  Same name images|
  folder.jpg|
* Cover Art Archive|
* Generic images|

Clicking on them would turn them on or of (* is here to symbol what's enabled), moving their order would change their priority.

The above example would mean for a given audio file, UMS would:
1. Look for embedded cover art
2. If 1) came up empty, try to look it up at Cover Art Archive
3. If 2) came up emtpy, use a generic image.

**Comment by SubJunk** [comment]
There is also a lot of potentially helpful discussion at https://github.com/UniversalMediaServer/UniversalMediaServer/issues/1360 about this
